packages/fxa-event-broker/types/mozlog/index.d.ts

```diff
@@ -0,0 +1,18 @@
+declare module 'mozlog' {
```

Comment:
I wonder if we can add this to the mozlog npm package?

Reply:
That'd definitely be preferable.